services/bundlejs/bundlejs-package.service.js

```diff
@@ -0,0 +1,130 @@
+import Joi from 'joi'
+import { BaseJsonService } from '../index.js'
+
+const schema = Joi.object({
+  size: Joi.object({
+    compressedSize: Joi.string(),
```

Comment:
```suggestion
    compressedSize: Joi.string().required(),
```

If we're going to use `compressedSize`, it needs to be `.required()`